style.css

```diff
@@ -0,0 +1,311 @@
+.first {
+  display: flex;
+  justify-content: space-between;
+}
+
+.header-first {
+  display: none;
+}
+
+.second {
+  background-image: url(./images/header-illsutration-mobile@2x.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  vertical-align: top;
+  text-align: center;
+  justify-content: center;
+  flex-direction: column;
+  font-weight: 700px;
+  font-size: 40px;
+  color: #172b4d;
+  padding: 10px 15px 15px 10px;
+  font-family: 'Inter', sans-serif;
+}
+
```

Comment:
- [ ] In 375px screens, your style doesn't seem to match the figma. Kindly consider using smaller fonts. Also, watch out your background image is at the right place.
> yours: 
![image](https://user-images.githubusercontent.com/91486791/205144360-72dc4b56-a963-4b73-a0b4-69646ac89284.png)
> figma: 
![image](https://user-images.githubusercontent.com/91486791/205144689-665c271e-38af-4541-9220-10d6ef3095fc.png)